### ISSUE TITLE
Cannot cast array data from dtype('int64') to dtype('int32') according to the rule 'safe'

### ISSUE THREAD

**Issue by BarbaL** [bytefish/facerec#57]
Hello Folks,

I ran this code many times in my Ubuntu 16.10 and got no problems. So, today i put the same code to run in a Raspberry Pi 3 Model B with Ubuntu MATE and when a face is detected in ./py/apps/videofacerec/simple_videofacerec.py the script stops and show the error message below:

```
Traceback (most recent call last):
  File "simple_videofacerec.py", line 232, in <module>
    cascade_filename=options.cascade_filename).run()
  File "simple_videofacerec.py", line 122, in run
    prediction = self.model.predict(face)[0]
  File "../../facerec/model.py", line 27, in predict
    return self.classifier.predict(q)
  File "../../facerec/classifier.py", line 95, in predict
    hist = dict((key,val) for key, val in enumerate(np.bincount(sorted_y)) if val)
TypeError: Cannot cast array data from dtype('int64') to dtype('int32') according to the rule 'safe'
```

In debug i found the error line, that says

`prediction = self.model.predict(face)[0]` on line 122.

Well, i know the architeture of RasPI 3 is different, is ARM architeture, but i don't think it is the problem. Someone know something about it? There's some incompatibility about 32bits processor?

**-- EDIT --**

Actually the processor of RasPI 3 B is 64 bits, but the OS i'm using is 32 bits. (Ubuntu MATE)

Thanks.
BarbaL

**Comment by bytefish** [comment]
It is almost the same error as in Issue #35. It is probably a problem on 32bit systems.

**Comment by bytefish** [comment]
I have added a fix in the duplicate ticket #65. Sorry for the massive delay.

Please reopen if the problem persists.
